src/btree/bt_handle.c

```diff
@@ -531,7 +531,11 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
      * connection-wide base write generation. In that case, we should initialize our write gen just
      * ahead of our btree specific write generation.
      */
-    btree->write_gen = btree->base_write_gen = WT_MAX(ckpt->write_gen + 1, conn->base_write_gen);
```

Comment:
Could you update the comment to be up to date? It might also warrant a TODO label linked to a JIRA ticket which tracks allowing dhandles to be closed while they have relevant history.

Reply:
5c129d6